configs/nvme-storage-configurator/base/install.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Enable debugging
+set -eux
+
+if [ -f /host-etc-systemd-dir/nvme-init-done ]; then
+    echo 'NVMe init already done'
+    oc adm uncordon $NODE_NAME
+    exit 0
+fi
+
+cp -f /scripts/mount.sh /host-usr-local-bin/mount.sh
+chmod 0755 /host-usr-local-bin/mount.sh
+
+cp -f /scripts/nvme-storage.service /host-etc-systemd-dir/nvme-storage.service
+chmod 0644 /host-etc-systemd-dir/nvme-storage.service
+
+oc adm cordon $NODE_NAME
+oc adm drain $NODE_NAME --delete-emptydir-data --ignore-daemonsets --grace-period=0 --force
```

Comment:
1. I think that we need to lt pods finish gracefully, other wise we would that pipelines fail because their pods are getting deleted.

2. what is the purpose of the --force flag? I didn't understand its description in the cli help message.

Reply:
removed --force flag and changed the grace-period to -1 to use the one coming from the pods.

Reply:
Now, Pods should be gracefully evicted.